packages/vuetify/src/components/VDialog/VDialog.sass

```diff
@@ -27,6 +27,9 @@
     > .v-card__text
       padding: 0 24px 20px
 
+    > .v-card__subtitle
+      padding: 0px 16px 16px 24px
```

Comment:
Any reason the right padding isn't also 24px to match the other elements? Bottom padding is also greater than with just a title.

Reply:
@KaelWD I might have set a wrong padding. I'll check later today and update my PR if necessary.